pkg/streamingpromql/testdata/upstream/functions.test

```diff
@@ -1329,21 +1327,17 @@ load 10s53ms
 	metric_histogram{type="only_histogram"} {{schema:1 sum:2 count:3}}x4
 	metric_histogram{type="mix"} 1 1 1 {{schema:1 sum:2 count:3}} {{schema:1 sum:2 count:3}} 1
 
-# Unsupported by streaming engine.
-# eval instant at 90s ts_of_last_over_time(metric[90s])
-#	{} 20.106
+eval instant at 90s ts_of_last_over_time(metric[90s])
+	{} 20.106
 
-# Unsupported by streaming engine.
-# eval instant at 95s ts_of_last_over_time(metric[90s])
-#	{} 20.106
+eval instant at 95s ts_of_last_over_time(metric[90s])
+	{} 20.106
 
-# Unsupported by streaming engine.
-# eval instant at 95s ts_of_last_over_time(metric_histogram{type="only_histogram"}[90s])
-# 	{type="only_histogram"} 40.212
+eval instant at 95s ts_of_last_over_time(metric_histogram{type="only_histogram"}[90s])
+	{type="only_histogram"} 40.212
 
-# Unsupported by streaming engine.
-# eval instant at 95s ts_of_last_over_time(metric_histogram{type="mix"}[90s])
-# 	{type="mix"} 50.265
+eval instant at 95s ts_of_last_over_time(metric_histogram{type="mix"}[90s])
+	{type="mix"} 50.265
```

Comment:
Could you please add a test for the case where there are both histograms and floats in the range and the histogram is the last point?